objects/d3f_technique.json

```
@@ -1,18 +1,22 @@
 {
   "caption": "MITRE DEFEND™ Technique",
-  "description": "The MITRE DEFEND™ Technique object describes the leaf defensive technique ID and/or name associated to a countermeasure, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND<sup>TM</sup> Matrix</a>.",
-  "extends": "_entity",
   "name": "d3f_technique",
+  "description": "The MITRE DEFEND™ Technique object describes the leaf defensive technique ID and/or name associated to a countermeasure.",
+  "references": [{"url": "href='https://d3fend.mitre.org", "description": "DEFEND™ Matrix"}],
+  "extends": "_entity",
   "attributes": {
     "name": {
-      "description": "The name of the defensive technique, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND<sup>TM</sup> Matrix</a>. For example: <code>IO Port Restriction</code>."
+      "description": "The name of the defensive technique. For example: <code>IO Port Restriction</code>.",
+      "references": [{"url": "https://d3fend.mitre.org", "description": "DEFEND™ Matrix"}]
     },
     "src_url": {
-      "description": "The versioned permalink of the defensive technique, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND<sup>TM</sup> Matrix</a>. For example: <code>https://d3fend.mitre.org/technique/d3f:IOPortRestriction/</code>.",
+      "description": "The versioned permalink of the defensive technique. For example: <code>https://d3fend.mitre.org/technique/d3f:IOPortRestriction/</code>.",
+      "references": [{"url": "https://d3fend.mitre.org", "description": "DEFEND™ Matrix"}],
       "requirement" : "optional"
     },
     "uid": {
-      "description": "The unique identifier of the defensive technique, as defined by <a target='_blank' href='https://mitre.mitre.org'>D3FEND<sup>TM</sup> Matrix</a>. For example: <code>D3-IOPR</code>."
+      "description": "The unique identifier of the defensive technique. For example: <code>D3-IOPR</code>.",
+      "references": [{"url": "https://d3fend.mitre.org", "description": "DEFEND™ Matrix"}]
```

```suggestion
      "references": [{"url": "https://d3fend.mitre.org", "description": "D3FEND™ Matrix"}]
```
Should be "D3FEND" (not "DEFEND").